### PR TITLE
main: issue regaring negative argument is now fixed

### PR DESCRIPTION
**Pull request by Amitava123**
Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Mentioned the issue number that it fixes
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Comment by 0xTheProDev** [comment]
**argv[1]** is a String or a sequence of characters terminated by _NUL ('\0')_
The comparison operator **<** expects numeric value in each side, not array or string!

File: main.c [hunk 1]
```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
 		printf("Usage: %s n\n", argv[0]);
 		return 1;
 	}
+	if(argv[1]<0){
```
Comment (RudraNilBasu) [inline main.c]: A space between the ")" and "{"

https://bytesclub.github.io/docs/coding_style.pdf

File: main.c [hunk 2]
```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
...
 	printf("%u", n);
 	
 	greet();
+        int row=0,col=0;
 	while(!f)
 	{
 		clear();
-		show(n, &n);
+                printf("\nPlayer 1: Enter Postion:\nRow Number: ");
+                row=getchar();
```
Comment (RudraNilBasu) [inline main.c:37]: A space before and after a an operator

File: main.c [hunk 3]
```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
...
+                printf("\nColumn Number: ");
+                col=getchar();
+		ARR[row][col]='X';
```
Comment (RudraNilBasu) [inline main.c:40]: Same space thing. + The name of the array needs to be changed. 
There might be a new issue targeting just on the coding structure

**Review by RudraNilBasu — COMMENTED** [review]
Besides the comments added by @Progyan1997 I have also added a few comments on the coding structure.

**Comment by 0xTheProDev** [comment]
The issue #13 has already been resolved. The PR is closing!

**Pull request closed without merging**